PMMain/Midas/MidasGUI/MyoTranslationFilter.cpp

```diff
@@ -66,6 +67,14 @@ void MyoTranslationFilter::process()
             }
         }
     }
+    else if (controlStateHandle->getMode() == KEYBOARD_MODE)
+    {
+        filterDataMap outputToSharedCommandData;
+        int mouseOrientationForKeyboard = keySelectAngle(orientation_data(roll, pitch, yaw));
+        outputToSharedCommandData[ORIENTATION_INPUT] = mouseOrientationForKeyboard;
```

Comment:
name should not have the word mouse in it haha. can you choose something more accurate plz